### DISCUSSION TITLE
Block references

### DISCUSSION THREAD

**Discussion by eyck-dev** [TriliumNext/Trilium#6023] [General]
Can we get block references like the one in logseq or obsidian. Referencing headers is a really easy way for students to write notes, without copying the same thing again and again.

**Comment by meichthys** [comment]
I'm not sure how Logseq or Obsidian work, but Trilium does have something similar called "Include Note".
You can think of every note as a 'block' and include notes inside of other notes. The 'included' note will update when the source note ('block') is updated:

![image](https://github.com/user-attachments/assets/774d5228-cd96-4370-8be9-93d03589a1a2)

![image](https://github.com/user-attachments/assets/c7bffee3-07d8-4eae-866f-3695454fc4fa)

**Comment by maphew** [comment]
It's a good idea that's been around for awhile, waiting for someone to get itchy enough to try and do something about it. ;-)
The enhancement issue is https://github.com/TriliumNext/Notes/issues/108

**Reply by meichthys** [reply]
I mis-understood the initial question - this is indeed a request for block-links rather than an way to include a block's content inside another note.

**Reply by eyck-dev** [reply]
Yes I meant this. Thanks for responding.

**Comment by CobriMediaJulien** [comment]
If we have enough people support this:

https://github.com/ckeditor/ckeditor5/issues/17264

Maybe CKEditor will implement this into the Editor. So lets support it. Check it out.
